### PR TITLE
Added autoFillOutForm in EntityBaseEditExtPage. That method works for…

### PR DESCRIPTION
**Pull request by homelyn-code**
… new and edit Entities.

### CODE REVIEW

File: src/test/java/runner/ProjectUtils.java [hunk 1]
```diff
@@ -84,6 +84,10 @@ public static String randomString(int stringLength){
         return createUUID();
     }
 
+    public static String randomString(){
+        return createUUID().substring(0, 10);
+    }
```
Comment (oooleks) [inline src/test/java/runner/ProjectUtils.java:89]: Why hardcode length = 10 if we have already randomString(int stringLength) and createUUID()

Comment (homelyn-code) [reply src/test/java/runner/ProjectUtils.java:89]: для удобства

Comment (oooleks) [reply src/test/java/runner/ProjectUtils.java:89]: so how user will now from its name that it's limited to 10 only, or rename it to randomTenSizeString or use randomString(10)

Comment (homelyn-code) [reply src/test/java/runner/ProjectUtils.java:100]: We have both methods, without params and with params

Comment (oooleks) [reply src/test/java/runner/ProjectUtils.java:100]: We've got already 3 methods to return random String: createUUID(); randomString(int stringLength) and getAlphanumeric(int count). So do you think we need one more with hardcoded length 10?

File: src/test/java/model/entity/edit/ExportEditPage.java [hunk 2]
```diff
@@ -43,13 +43,13 @@ protected ExportEditPage createEditPage() {
...
-        ProjectUtils.click(driver, saveButton);
-        return createPage();
-    }
+//    @Override
+//    public ExportPage clickSaveButton() {
+//        WebDriver driver = getDriver();
+//        WebElement saveButton = driver.findElement(BY_SAVE_BUTTON);
+//        ProjectUtils.click(driver, saveButton);
+//        return createPage();
+//    }
```
Comment (oooleks) [inline src/test/java/model/entity/edit/ExportEditPage.java:52]: delete commented code

**Review by oooleks — CHANGES_REQUESTED** [review]

File: src/test/java/runner/ProjectUtils.java (hunk 1 above)
Comment (homelyn-code) [reply src/test/java/runner/ProjectUtils.java:100]: Yep, we can remove createUUID()

**Review by oooleks — APPROVED** [review]

**Review by guram21 — APPROVED** [review]

**Pull request closed without merging**